index.html

```diff
@@ -0,0 +1,228 @@
+                            </ul>
+                        </li>
+                        <li class="category fram">Framework
+                            <ul class="sublist">
+                                <li>Bootstrap</li>
+                                <li>Ruby on Rails</li>
+                                <li>RSPec</li>
+                                <li>CapyBara</li>
+                                <li>Selenium</li>
+                            </ul>
+                        </li>
+                        <li class="category skills">Skills
+                            <ul class="sublist">
+                                <li>Codekit</li>
+                                <li>Github</li>
+                                <li>Codepen</li>
+                                <li>Gitlab</li>
+                                <li>Terminal</li>
+                            </ul>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </section>
+        <footer class="footer">
+            <h3 class="footerh3">I'm always <br>interested in hearing about new projects,<br> so if you'd like to chat please get in <br>touch.</h3>
+            <form id="frmContact" class="form-container" action="https://formspree.io/f/xoqbgwkd" method="post">
+                <label for="name" hidden>Username input</label>
+                <input class="form-name" type="text" name="name" placeholder="Full name" maxlength="30" required>
+          
+                <label for="email" hidden>Email input</label>
+                <input class="form-email" type="email" name="email" placeholder="Email adress" required>
+          
+                <label for="message" hidden>Comment area</label>
+                <textarea class="form-comment" name="message" maxlength="500" rows="10" placeholder="Write me something..." required></textarea>
+          
+                <button class="btn" type="submit">Get in touch</button>
+            </form>
+            <div class="iconsss">
+                <ul class="icon-bar extra-bar">
+                    <li class="icontag2">
+                    <a href="https://twitter.com/tamoorsaeed9"><img src="images/twitter.png" alt="Twitter"></a>
+                 </li>
+                    <li class="icontag2">
+                    <a href="https://www.linkedin.com/in/tamoor-saeed-58912a233/"><img src="images/Linkedin.png" alt="Linkedin"></a>
+                    </li>
+                    <li class="icontag2">
+                    <a href="https://github.com/tamoorsaeed22"><img src="images/github.png" alt="Github"></a>
+                    </li>
+                    <li class="icontag2">
+                    <a href=""><img src="images/Angellist.png" alt="Angellist"></a>
+                    </li>
+                    <li class="icontag2">
+                    <a href=""><img src="images/Medium.png" alt="Medium"></a>
+                    </li>
+                </ul>
+            </div>
```

Comment:
**OPTIONAL** - To help improve the semantics of your site, I suggest that you wrap the contact section in a `section` tag instead of a `footer` tag. Pleasefix this 😄 

- According to the project requirements, your project should be responsive for all mobile screens i.e screens between `375px and 768px`. Your project (contact section), mandatory and optionally(other parts of your project) is not responsive. Please fix this 🙏 

Your project:
<img width="557" alt="fbsfsfb" src="https://user-images.githubusercontent.com/73607512/228013310-1d5038d3-849a-4dab-aa32-db436f53a901.PNG">

Reply:
@mckent05 according to the project requirements, your project should be responsive for all mobile screens i.e screens between 375px and 768px have been resolved and responsive.

Reply:
> @mckent05 according to the project requirements, your project should be responsive for all mobile screens i.e screens between 375px and 768px have been resolved and responsive.

- [ ] Just like the previous reviewer mention you design is meant to be responsive for screens between 375px and 768px, you can achieve this by using `media-queries` in css, check this [link](https://www.youtube.com/watch?v=yU7jJ3NbPdA) for more.